nano/core_test/enums.cpp

```diff
@@ -7,17 +7,59 @@
 TEST (enums, stat_type)
 {
 	ASSERT_FALSE (nano::to_string (static_cast<nano::stat::type> (0)).empty ());
+	ASSERT_NO_THROW (std::string{ nano::to_string (static_cast<nano::stat::type> (0)) });
+
 	ASSERT_FALSE (nano::to_string (nano::stat::type::_last).empty ());
+	ASSERT_NO_THROW (std::string{ nano::to_string (nano::stat::type::_last) });
+	ASSERT_EQ (nano::to_string (nano::stat::type::_last), "_last");
```

Comment:
Doesn't this line do the job of the both lines above it?

Reply:
What happens if `nano::to_string (nano::stat::type::_last)` returns a malformed string view with size set to a random value? This seems to be fixed in the latest magic enum version but was a problem before. If assert_eq fails it's most likely going to print this large, malformed string, here it will get caught sooner. Also, I added those lines so they follow a pattern.